### PR TITLE
The loss do not go down with original learning rate

### PR DESCRIPTION
**Pull request by wangdelp**
1. reduced the learning rate to 0.001
100 epoch val_loss: 0.46, val_acc: 0.84
200 epoch val_loss: 0.385, val_acc: 0.871

2. add config for tensorflow backend to prevent it from comsuming all the GPU memories

### CODE REVIEW

**Comment by fchollet** [comment]
It definitely does.

**Comment by patyork** [comment]
I wouldn't dismiss these issues out of hand (although this should have been an issue first and not a pull request without proof).

OOTB script, Theano backend, first 35 epochs, is below. I have yet to get an example to converge or get close to the stated results from the docstrings where the model is using SGD.
```
3X_train shape: (50000, 32, 32, 3)
50000 train samples
10000 test samples
Using real-time data augmentation.
Epoch 1/200 50000/50000 - 55s - loss: 2.3037 - acc: 0.0978 - val_loss: 2.3035 - val_acc: 0.1000
Epoch 2/200 50000/50000 - 55s - loss: 2.3034 - acc: 0.0995 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 3/200 50000/50000 - 55s - loss: 2.3032 - acc: 0.0999 - val_loss: 2.3034 - val_acc: 0.1000
Epoch 4/200 50000/50000 - 55s - loss: 2.3034 - acc: 0.0975 - val_loss: 2.3034 - val_acc: 0.1000
Epoch 5/200 50000/50000 - 55s - loss: 2.3033 - acc: 0.0995 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 6/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0976 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 7/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0990 - val_loss: 2.3036 - val_acc: 0.1000
Epoch 8/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0980 - val_loss: 2.3034 - val_acc: 0.1000
Epoch 9/200 50000/50000 - 56s - loss: 2.3035 - acc: 0.0993 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 10/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0971 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 11/200 50000/50000 - 56s - loss: 2.3035 - acc: 0.0978 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 12/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0995 - val_loss: 2.3036 - val_acc: 0.1000
Epoch 13/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.1001 - val_loss: 2.3031 - val_acc: 0.1000
Epoch 14/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0988 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 15/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0995 - val_loss: 2.3036 - val_acc: 0.1000
Epoch 16/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0987 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 17/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.1008 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 18/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0988 - val_loss: 2.3031 - val_acc: 0.1000
Epoch 19/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.1009 - val_loss: 2.3028 - val_acc: 0.1000
Epoch 20/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0989 - val_loss: 2.3028 - val_acc: 0.1000
Epoch 21/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.1003 - val_loss: 2.3035 - val_acc: 0.1000
Epoch 22/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0983 - val_loss: 2.3027 - val_acc: 0.1000
Epoch 23/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0987 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 24/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.1008 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 25/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0973 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 26/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0984 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 27/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0974 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 28/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0970 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 29/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0988 - val_loss: 2.3034 - val_acc: 0.1000
Epoch 30/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0991 - val_loss: 2.3026 - val_acc: 0.1000
Epoch 31/200 50000/50000 - 56s - loss: 2.3034 - acc: 0.0986 - val_loss: 2.3028 - val_acc: 0.1000
Epoch 32/200 50000/50000 - 56s - loss: 2.3032 - acc: 0.1003 - val_loss: 2.3034 - val_acc: 0.1000
Epoch 33/200 50000/50000 - 56s - loss: 2.3032 - acc: 0.1010 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 34/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.1005 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 35/200 50000/50000 - 56s - loss: 2.3033 - acc: 0.0994 - val_loss: 2.3035 - val_acc: 0.1000
```

**Comment by wangdelp** [comment]
@patyork Thanks, I got the same result using Tensorflow backend with the original learning rate.

**Comment by patyork** [comment]
@wangdelp Yeah, this should have been brought up as an issue first. It has been, multiple times though (#4587, #4066). The solution has always been to use an optimizer other than SGD (or, change the LR of the SGD), but no one has changed the example to do this, in order to solve the problem once and for all.

It seems, to me at least, to be a larger issue though. It seems like something has changed with the SGD implementation to where every example that uses SGD as an optimizer no longer learns, or learns as well as the docstring states.
